pkg/cluster/internal/kubeadm/config.go

```diff
@@ -140,6 +144,8 @@ apiServerExtraVolumes:
 apiServerCertSANs: [localhost]
 controllerManagerExtraArgs:
   enable-hostpath-provisioner: "true"
+networking:
+  podSubnet: {{ .PodSubnet }}
```

Comment:
I had issues when using IPv6 addresses, we should quote it
`podSubnet: "{{ .PodSubnet }}"`

Reply:
done